packages/perseus/src/widgets/interactive-graphs/graphs/components/angle.tsx

```diff
@@ -123,6 +134,20 @@ export const Angle = ({
     );
 };
 
+const RightAngleArc = ({start: [x1, y1], vertex: [x2, y2], end: [x3, y3]}) => (
```

Comment:
Suggestion: should this be called `RightAngleSquare`, since it's not really an arc?